org.kde.ghostwriter.yml

```diff
@@ -0,0 +1,58 @@
+app-id: org.kde.ghostwriter
+runtime: org.kde.Platform
+runtime-version: 5.15-22.08
+base: io.qt.qtwebengine.BaseApp
```

Comment:
Cleanup baseapp https://github.com/flathub/io.qt.qtwebengine.BaseApp/wiki

Reply:
Ping for cleanup.